.github/workflows/nodejs.yml

```diff
@@ -5,18 +5,23 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master ]
+    tags:
+      - '*'
+    branches:
+      - '*'
   pull_request:
-    branches: [ master ]
+    branches:
```

Comment:
Allow CI to run on every pull request. This can help catch failing tests/lint issues before the code gets merged.